### PR TITLE
Merge master to dev

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**

### CODE REVIEW

File: common/versioning/version.properties [hunk 1]
```diff
@@ -1,4 +1,9 @@
+<<<<<<< HEAD
 #Thu May 06 17:06:14 UTC 2021
 versionName=3.4.2
+=======
+#Tue Apr 06 22:55:08 UTC 2021
+versionName=3.4.3
+>>>>>>> master
```
Comment (AdamBJohnsonx) [inline common/versioning/version.properties:7]:
```suggestion
#Tue Apr 06 22:55:08 UTC 2021
versionName=3.4.3
```

Comment (shahzaibj) [inline common/versioning/version.properties:7]: Needs cleanup

Comment (AdamBJohnsonx) [reply common/versioning/version.properties]: already on it

File: common/versioning/version.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#Thu May 06 17:06:14 UTC 2021
-versionName=3.4.2
+#Tue Apr 06 22:55:08 UTC 2021
```
Comment (AdamBJohnsonx) [inline common/versioning/version.properties:1]:
```suggestion
#Thu May 06 17:06:14 UTC 2021
```

**Review by iambmelt — APPROVED** [review]

**Review by shahzaibj — APPROVED** [review]

**Pull request merged**